scripts/index.js

```diff
@@ -429,30 +432,43 @@ program
         startWebsite();
     });
 
+program
+    .command('dev')
+    .description('watch for changes in `src` and `test` directories')
+    .action(() => {
+        cp.spawn('npm run build -- -f -w', { stdio: 'inherit', shell: true });
+        cp.spawn('npm run build-tests -- -w', { stdio: 'inherit', shell: true });
+    });
+
 program
     .command('build')
     .description('build dist')
     .option('-f, --fast', 'skip minifying')
     .option('-w, --watch')
-    .option('-d, --dest [path]', 'specify the build destination path')
-    .option('-x, --exclude [exclude...]')
-    .option('-m, --modules [modules...]')
+    .option('-i, --input <...path>', 'specify the build input paths')
+    .option('-o, --output <path>', 'specify the build output path')
+    .option('-x, --exclude <exclude...>')
+    .option('-m, --modules <modules...>')
     .action((options) => {
-        const { dest: destination, ...rest } = options || {};
-        build({ ...rest, destination });
+        build(options);
     });
 
 program
     .command('test')
     .description('run test suite')
-    .addOption(new commander.Option('-s, --suite [suite...]', 'test suite to run').choices(['unit', 'visual']))
-    .option('-f, --file [file]', 'run a specific test file')
-    .option('--filter [filter]', 'filter tests by name')
+    .addOption(new commander.Option('-s, --suite <suite...>', 'test suite to run').choices(['unit', 'visual']))
+    .option('-f, --file <file>', 'run a specific test file')
+    .option('--filter <filter>', 'filter tests by name')
     .option('-a, --all', 'run all tests', false)
     .option('-d, --debug', 'debug visual tests by overriding refs (golden images) in case of visual changes', false)
     .option('-r, --recreate', 'recreate visual refs (golden images)', false)
     .option('-v, --verbose', 'log passing tests', false)
-    .option('-cc, --clear-cache', 'clear CLI test cache', false)
+    .option('-l, --launch', 'launch tests in the browser', false)
+    .option('--dev', 'runs testem in `dev` mode, without a `ci` flag', false)
+    .addOption(new commander.Option('-c, --context <context...>', 'context to test in').choices(['chrome', 'firefox', 'node']).default(['chrome', 'node']))
+    .option('-p, --port')
+    .option('-o, --out <out>', 'path to report test results to')
+    .option('--clear-cache', 'clear CLI test cache', false)
```

Comment:
You should add the option `--coverage` here

Reply:
https://github.com/testem/testem#running-browser-code-after-tests-complete

Reply:
karma has it....
https://www.npmjs.com/package/karma-coverage

Reply:
https://github.com/testem/testem/blob/master/examples/coverage_istanbul/

Reply:
![image](https://user-images.githubusercontent.com/34343793/185954569-e7184fa3-c9b2-40f7-baa0-b6fbf5d7d5f4.png)
I managed to get it to work only on `node`
is that enough?

Reply:
yes coverage needs only node, but results are weird in this screenshot.

Reply:
it's only for activeselection